src/commands/deploy/latest.js

```diff
@@ -668,38 +668,48 @@ function handleCreateDeployError(output, error) {
     return 1;
   }
   if (error instanceof SchemaValidationFailed) {
-    const { params, keyword, dataPath } = error.meta;
+    const { message, params, keyword, dataPath } = error.meta;
+
     if (params && params.additionalProperty) {
       const prop = params.additionalProperty;
+
       output.error(
         `The property ${code(prop)} is not allowed in ${highlight(
           'now.json'
         )} when using Now 2.0 – please remove it.`
       );
+
       if (prop === 'build.env' || prop === 'builds.env') {
         output.note(
           `Do you mean ${code('build')} (object) with a property ${code(
             'env'
           )} (object) instead of ${code(prop)}?`
         );
       }
+
       return 1;
     }
+
     if (keyword === 'type') {
       const prop = dataPath.substr(1, dataPath.length);
+
       output.error(
         `The property ${code(prop)} in ${highlight(
           'now.json'
         )} can only be of type ${code(title(params.type))}.`
       );
+
       return 1;
     }
+
     const link = 'https://zeit.co/docs/v2/deployments/configuration/';
+
     output.error(
       `Failed to validate ${highlight(
         'now.json'
-      )}. Only use properties mentioned here: ${link}`
+      )}: ${message}\nFull list of allowed properties: ${link}`
```

Comment:
`Docs: ` imo. Allowed properties being singled out makes it sound like the customer used a disallowed  prop

Reply:
Great call. Fixed!